documentation/release_notes.rst

```diff
@@ -53,6 +53,8 @@ New in This Release
   with ``unseq`` or ``par_unseq`` policy when compiled by Intel® oneAPI DPC++/C++ Compiler
   with ``-fiopenmp``, ``-fiopenmp-simd``, ``-qopenmp``, ``-qopenmp-simd`` options on Linux.
   To avoid the issue, pass ``-fopenmp`` or ``-fopenmp-simd`` option instead.
+- Incorrect results may be produced with group reduce operation through ``reduce``, ``transform_reduce``
+  for 64-bit data types and ``std::multiplies``, ``sycl::multiplies`` operations on GPUs with the Intel® C++ Compiler 2021.3
```

Comment:
```suggestion
- Incorrect results may be produced by ``reduce`` and ``transform_reduce`` with 64-bit types and ``std::multiplies``, ``sycl::multiplies`` operations when compiled on Intel® C++ Compiler 2021.3 and newer and executed on GPU devices. 
```
Changes:
 - Removed "with group reduce operation" because it is an internal implementation detail.
 - Added "... and newer" to highlight that the issue arises not only on Intel® C++ Compiler 2021.3, but also newer version.
 - Removed "data" as redundant
 - Fixed a type and slightly restructured the note.